doc/installation.rst

```diff
@@ -131,112 +140,59 @@ Installing with pip (experts only)
-
-* `Pylint <http://pylint.pycqa.org/>`_  -- for static code analysis.
-
-* `Psutil <https://github.com/giampaolo/psutil>`_  -- for memory/CPU usage in the status bar.
-
-* `Nbconvert <https://github.com/jupyter/nbconvert>`_ -- to manipulate Jupyter notebooks on the Editor.
-
-* `Qtawesome <https://github.com/spyder-ide/qtawesome>`_ >=0.4.1 -- for an icon theme based on FontAwesome.
-
-* `Pickleshare <https://github.com/pickleshare/pickleshare>`_ -- To show import completions in the Editor and Consoles.
-
-* `PyZMQ <https://github.com/zeromq/pyzmq>`_ -- CLient for the Language Server Protocol.
-
-* `QtPy <https://github.com/spyder-ide/qtpy>`_ >=1.2.0 -- To run Spyder with different Qt bindings seamlessly.
-
-* `Chardet <https://github.com/chardet/chardet>`_ >=2.0.0-- Character encoding auto-detection in the Editor.
-
-* `Numpydoc <https://github.com/numpy/numpydoc>`_ Used by Jedi to get return types for functions with Numpydoc docstrings.
-
-* `Cloudpickle <https://github.com/cloudpipe/cloudpickle>`_ Serialize variables in the IPython kernel to send them to Spyder.
-
-
-Optional modules
-~~~~~~~~~~~~~~~~
-
-* `Matplotlib <https://matplotlib.org/>`_ >=1.0 -- for 2D and 3D plotting in the consoles.
-
-* `Pandas <https://pandas.pydata.org/>`_ >=0.13.1 -- for viewing and editing Series and DataFrames in the Variable Explorer.
-
-* `Numpy <https://www.numpy.org/>`_ -- for viewing and editing two or three dimensional arrays in the Variable Explorer.
-
-* `Sympy <https://www.sympy.org/en/index.html>`_ >=0.7.3 -- for working with symbolic mathematics in the IPython console.
-
-* `Scipy <https://www.scipy.org/>`_ -- for importing Matlab workspace files in the Variable Explorer.
-
-* `Cython <http://cython.org/>`_ >=0.21 -- to run Cython files or Python files that depend on Cython libraries in the IPython console.
-
-
-Installation procedure
-~~~~~~~~~~~~~~~~~~~~~~
-
 You can install Spyder with the ``pip`` package manager, which comes by default with most Python installations.
 Before installing Spyder itself by this method, you need to acquire the `Python`_ programming language.
 
 .. _Python: https://www.python.org/
 
-Then, to install Spyder and its other dependencies, run ``pip install spyder``.
-You may need to install a Qt binding (PyQt5) separately with ``pip`` if running under Python 2.
-
-
-Run without installing
-~~~~~~~~~~~~~~~~~~~~~~
+You'll first want to create and activate a virtual environment in which to install Spyder, via one of the following methods.
 
-You can execute Spyder from source without installing it first by the following procedure:
+With `virtualenvwrapper`:
 
-#. Unzip the source package available for download on the `Spyder Github repository`_ (or :ref:`clone it from Github<install-from-source>`)
-#. Change current directory to the unzipped directory
-#. Install Spyder's requirements with:
+```bash
```

Comment:
Please fix the code blocks

Reply:
Done, thanks for the reminder. There were quite a few I didn't have right.